include/seastar/core/future.hh

```diff
@@ -1952,14 +1952,14 @@ template<typename T>
 template<typename Func, typename... FuncArgs>
 typename futurize<T>::type futurize<T>::invoke(Func&& func, FuncArgs&&... args) noexcept {
```

Comment:
>  - Forward the function itself, not just the arguments. This allows
   an rvalue-qualified operator() to be called if present.

950b54ef3d3fd8205d5424dc4bdae94f8573bb25 beat you to it, so now you get to rebase and I get to re-run CI